backend/src/main/java/ca/bc/gov/hlth/mohums/controller/DashboardController.java

```diff
@@ -0,0 +1,56 @@
+package ca.bc.gov.hlth.mohums.controller;
+
+import java.util.List;
+import java.util.Map;
+import java.sql.*;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class DashboardController {
+
+    @Value("${spring.datasource.url}")
+    private String url;
+
+    @Value("${spring.datasource.username}")
+    private String username;
+
+    @Value("${spring.datasource.password}")
+    private String password;
+
+    @Autowired
+    private JdbcTemplate jdbcTemplate;
+
+    @GetMapping("/dashboard/active-user-count")
+    public List<Map<String, Object>> getActiveUserCount() throws SQLException {
+        String sql = "SELECT realm_id AS REALM, client_id AS CLIENT, COUNT(1) AS ACTIVE_USER_COUNT FROM ( SELECT DISTINCT ue.realm_id, c.client_id, ue.username FROM event_entity ee INNER JOIN user_entity ue ON ue.id = ee.user_id INNER JOIN client c ON c.client_id = ee.client_id INNER JOIN keycloak_role kcr ON kcr.client = c.id INNER JOIN user_role_mapping urm ON urm.user_id = ee.user_id AND urm.role_id = kcr.id WHERE ee.type = 'LOGIN' AND ee.event_time > (SYSDATE-365-TO_DATE('1970-01-01','YYYY-MM-DD'))*24*60*60 AND ue.enabled = 1 UNION SELECT DISTINCT ue.realm_id, c.client_id, ue.username FROM event_entity ee INNER JOIN user_entity ue ON ue.id = ee.user_id INNER JOIN client c ON c.client_id = ee.client_id LEFT OUTER JOIN keycloak_role kcr ON kcr.client = c.id WHERE ee.type = 'LOGIN' AND ee.event_time > (SYSDATE-365-TO_DATE('1970-01-01','YYYY-MM-DD'))*24*60*60 AND ue.enabled = 1  AND kcr.id IS NULL ) WHERE NOT (client_id = 'account' AND (realm_id IN ('bceid_basic', 'bceid_business', 'idir', 'idir_aad', 'master', 'moh_applications', 'moh_citizen', 'phsa')) OR (client_id IN ('realm-management', 'USER-MANAGEMENT-SERVICE', 'PRIME-WEBAPP-ENROLLMENT'))) GROUP BY realm_id, client_id  ORDER BY realm_id ASC, client_id ASC";
```

Comment:
I know you mentioned this earlier, but there must be something wrong with this query. `davidscoolnewrealm` is cool, but  I think it was deleted a long time ago. It doesn't show up in the Keycloak Admin Console anymore. Maybe the Keycloak DB maintains a record of it or something, but we should filter out deleted realms.

The data is also surprising because I would think someone has logged into DEV PLR within the past 365 days, and maybe others, like FMDB.

![image](https://user-images.githubusercontent.com/1767127/170368577-ef319efc-5965-4ef0-af71-82d3095cdf6b.png)

Reply:
I talked to adam about this, and he said that the query works fine. He doubled checked and davidscoolnewrealm still exists and is enabled. 

as for the surprising data, he also mentioned that FMDB being omitted made sense because it hasn't been touched for a couple years.

Reply:
resolved. davidscoolnewrealm is the ID of another realm(different name).